Gemfile

```
@@ -8,3 +8,4 @@ gemspec
 gem "benchmark-ips"
 gem "pry-byebug"
 gem "activesupport"
+gem "steep"
```

```suggestion
gem "steep", require: false
```

To fix this failure

https://github.com/mtsmfm/language_server-protocol-ruby/actions/runs/9206138738/job/25323641047

I know it's already retired though I'd like to not drop old rubies until we face huge difficulties just for my curiosity

---

@joker1007 Can you apply this change?

---

@joker1007 Poke

---

I'm sorry. I forgot to apply the change. 
I fixed it.